### DISCUSSION TITLE
Job freshness

### DISCUSSION THREAD

**Discussion by cuihtlauac** [ocaml/ocaml.org#1286] [General]
Jobs should have a freshness label such as (not all options make sense, this is just illustrative):
- “Posted today”
- “Posted X days ago”
- “Posted this week”
- “Posted X weeks ago”
- “Posted this month”
- “Posted X months ago“

However, there are two special/special cases:
- “Evergreen” (such as discussed in https://github.com/ocaml/ocaml.org/pull/909)
- “Freshness status unknown”

**Comment by sabine** [comment]
I think this is closely related to categorizing jobs and how we deal with new incoming listings and what sorting and filtering options there will be.

IMO the existing publication date is fine as a freshness label, but it would be helpful to establish categories so it is clear who the job is addressed to.

E.g.
- category of job (employee vs. contractor/freelancer)
- expectation on the worker's time commitment (part-time, full-time)
- duration of the job (short-term vs. long-term)
- type of listing (spontaneous application vs a specific job posting)

:thinking: Something like that.
